README.md

```diff
@@ -81,19 +87,15 @@ handling logic.
 	package main
 
 	import (
-		"fmt"
 		"log"
-		"net/http"
 		"os"
 
-		"cloud.google.com/go/functions/framework"
+		"github.com/GoogleCloudPlatform/functions-framework-go/framework"
 		"example.com/hello"
 	)
 
 	func main() {
-		if err := framework.RegisterHTTPFunction("/", hello.HelloWorld); err != nil {
-			log.Fatalf("framework.RegisterHTTPFunction: %v\n", err)
-		}
+		framework.RegisterHTTPFunction("/", hello.HelloWorld); err != nil {
```

Comment:
This sample does not build.

` err != nil {` does not look right here.

```
./main.go:14:14: syntax error: unexpected :=, expecting comma or }
./main.go:15:9: syntax error: unexpected if at end of statement
```

Are we sure that this was tested?

Reply:
Yeah but I updated the wrong file. My bad!